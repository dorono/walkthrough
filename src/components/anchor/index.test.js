import React from 'react';
import {shallow} from 'enzyme';
import Anchor from './index';

describe('Anchor', () => {
    it('should render without errors', () => {
        shallow(<Anchor />);
    });

    it('should match snapshot', () => {
        expect(shallow(<Anchor />)).toMatchSnapshot();
    });
});
