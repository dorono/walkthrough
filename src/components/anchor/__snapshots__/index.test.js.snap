// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Anchor should match snapshot 1`] = `
<a
  href="#"
  onClick={[Function]}
  target="_blank"
/>
`;
